Sources/SourceKitLSP/Swift/RewriteSourceKitPlaceholders.swift

```diff
@@ -134,8 +134,8 @@ private func tokenize(_ input: String) -> [SnippetToken] {
 private enum SnippetToken {
   /// A placeholder delimiter.
   case placeholderOpen, placeholderClose
-  /// One of '{' or '}', which may need to be escaped in the output.
-  case curlyBrace(Character)
+  /// '}', which may need to be escaped in the output.
```

Comment:
Argh, yup, this was my mistake, @ahoppen; the LSP spec doesn't require escaping of open braces. This change works fine for me. (My editor is Emacs, using the eglot LSP client.)

Related note; `$` and `\` are also supposed to be escaped; I don't know if you think it's worth adding those too. I would be happy to do it myself as well.

Reply:
Thanks for checking with emacs and noticing that we also need to escape `$` and `\`. I missed that in the LSP the first time I looked for a definition of the escaping rules.